inst/include/intermediate_data.hpp

```diff
@@ -245,6 +245,10 @@ struct ChildModelIntermediateData<ChildModel, real_type> {
   TensorFixedSize <real_type, Sizes<4>> hc_initByAge;
   TensorFixedSize <real_type, Sizes<4>> hc_adj;
   TensorFixedSize <real_type, Sizes<4>> hc_art_deaths;
+  //TODO: change this hard coding
+  TensorFixedSize <real_type, Sizes<hDS<ChildModel>, hcTT<ChildModel>, 15, NS<ChildModel>>> hc_hiv_dist;
+  TensorFixedSize <real_type, Sizes<hDS<ChildModel>,  15, NS<ChildModel>>> hc_hiv_total;
+  TensorFixedSize <real_type, Sizes<hDS<ChildModel>, hcTT<ChildModel>, 15, NS<ChildModel>>> art_ltfu_grad;
```

Comment:
Do you want to address this hard coding here?

Reply:
fixed!